metadata-ingestion/src/datahub/ingestion/source/identity/okta.py

```diff
@@ -407,6 +445,14 @@ def get_workunits(self) -> Iterable[MetadataWorkUnit]:
         # Step 4: Close the event loop
         event_loop.close()
 
+    def get_workunits(self) -> Iterable[MetadataWorkUnit]:
+        return auto_stale_entity_removal(
+            self.stale_entity_removal_handler,
+            auto_workunit_reporter(
```

Comment:
The auto_workunit_reporter is not appropriate here, but the other two helpers are correct

Reply:
done